### DISCUSSION TITLE
clang-format not working

### DISCUSSION THREAD

**Discussion by prabanjanraja** [LazyVim/LazyVim#2082] [Q&A]
the cpp files are getting formatted but they do not follow the rules defined in hte .clang-format file in the root of the directory
find the 
please find the details of the lsp attached 
![Untitled](https://github.com/LazyVim/LazyVim/assets/56388091/3cc46ed3-4762-4ad2-9569-c760ae3347ca)

**Comment by jcma86** [comment]
Did you find a solution?
